pkgs/os-specific/linux/usbguard/default.nix

```diff
@@ -12,14 +12,14 @@ with stdenv.lib;
 assert libgcrypt != null -> libsodium == null;
 
 stdenv.mkDerivation rec {
-  version = "0.7.8";
+  version = "1.0.0";
   pname = "usbguard";
 
   repo = "https://github.com/USBGuard/usbguard";
 
   src = fetchurl {
     url = "${repo}/releases/download/${pname}-${version}/${pname}-${version}.tar.gz";
-    sha256 = "1il5immqfxh2cj8wn1bfk7l42inflzgjf07yqprpz7r3lalbxc25";
+    sha256 = "sha256-VheYbNXdGi0xEEFkihl32DbPTjOkEh1/glmfIUlqvEI=";
   };
```

Comment:
Can't we use `fetchFromGithub` instead :thinking: ?

Reply:
That's what I considered but it needs the `tar.gz` release rather than the `tar.gz` source

If it used the source code it'd need to run `./autogen.sh` which gets submodules and autoreconf
https://github.com/USBGuard/usbguard#compilation-installation

I guess `fetchFromGithub`/the just git one can grab submodules and `autoreconf` isn't too hard to run so I'll have a go soon

Reply:
I probably wont get to it until tomorrow morning europe time so maybe it's better to merge this this PR separate as it is now and add it in another?

Reply:
We are on no rush here and have the time to wait one day to build this from source.

Reply:
Ok it was easier than expected. `autoreconfHook` seems to have handled things fine without the need for a `preAutoreconf` phase